tests/test_optical.py

```diff
@@ -1,5 +1,111 @@
-from optical import __version__
+from pathlib import Path
 
+import random
+import pytest
 
-def test_version():
-    assert __version__ == "0.1.0"
+from optical.converter.utils import find_splits
+from optical import Annotation
+from optical.converter import FormatSpec
+from optical.converter.converter import (  # noqa: F401
+    convert_coco,
+    convert_yolo,
+    convert_csv,
+    convert_pascal,
+    convert_createml,
+    convert_sagemaker,
+    convert_tfrecord,
+    convert_simple_json,
+)
+
+import pandas as pd
+import altair as alt
+
```

Comment:
I think we need to check tensorflow installation, before expecting it while running test case